### PR TITLE
EffectType extending.

### PR DESCRIPTION
**Pull request by svtv**
Attempt to extend the Raid.DataModel.EffectType with missed data. All properties declared in the SharedModel.Battle.Effects.EffectType have to be read now. Kindly appreciate any comments.

### CODE REVIEW

File: SDK/Service/Facets/StaticData/StaticDataFacet.cs [hunk 1]
```diff
@@ -24,7 +24,10 @@ protected override StaticData Merge(ModelScope scope, StaticData previous = null
                 kvp => (DataModel.Enums.HeroRarity)kvp.Key,
                 kvp => kvp.Value.Select(bonuses => bonuses.Bonuses.Select(bonus => bonus.Value.ToModel(bonus.Key)).ToArray()).ToArray()
             );
-            var localizedStrings = new Dictionary<string, string>(staticData.ClientLocalization.Concat(staticData.StaticDataLocalization));
+            var localizedStrings = new Dictionary<string, string>(staticData.ClientLocalization
```
Comment (dnchattan) [inline SDK/Service/Facets/StaticData/StaticDataFacet.cs:27]: I'm guessing this is an unrelated fix for the recent break with string key collisions?

**Comment by dnchattan** [comment]
Thanks a ton for submitting a PR!
Can you share a little bit about what your use case is for this data?

When possible I kind of want to avoid duplicating the full raw raid model in RTK types if possible, simply because it's a bit of a maintenance task to keep them up to date as the game changes. To be clear, I'm not opposed to this change, but want to make sure I understand what you're aiming for in case there is a different way it might fit in 😀


Also, as the you're the first community member to submit a PR: any feedback on getting up and running/building/testing with the project? Anywhere it could be improved (especially in documentation) for anyone else wanting to jump in and contribute?

**Comment by dnchattan** [comment]
Also @LukeCroteau would love your thoughts on this at a high level.

File: SDK/Service/Facets/StaticData/StaticDataFacet.cs (hunk 1 above)
Comment (dnchattan) [reply SDK/Service/Facets/StaticData/StaticDataFacet.cs:27]: (I was just fixing this same issue, so I'll probably pull this in separately right now as a patch fix for 1.3 and 1.4 :smile:)

**Comment by svtv** [comment]
> Thanks a lot, it is pleasure to be a community member.
My use case for this data is quite simple. As a novice in the game, I want to understand the game mechanics as well as to make a tool that would help other game novices to start better.
Like you, I'm not very happy with tons of these stubs. But the alternative is copying the Raid.Interop.dll to all locations where it would be used. Not the best way either.
Anyhow, I'm open to any ideas and would be glad to take part in developing this great tool.

**Review by dnchattan — APPROVED** [review]
